.github/pull_request_template.md

```diff
@@ -4,7 +4,7 @@ Thanks for your contribution. It must be reviewed by a member of the [`cockroach
 - [ ] Successful local linkcheck (`make linkcheck`)
 - [ ] Successful pull request tests
 
-To view a HTML preview of each modified page, look for the **Files Changed** comment from the **github-actions** bot. Unfortunately, previews are not available if you use a fork.
+To view a HTML preview of each modified page, look for the **Netlify Preview** comment from the **netlify** bot.
```

Comment:
See, I thought it was Netlify that didn't run on forks. Netlify doesn't run if your PR is against a non-fork branch besides `master`. Right? Why can't we run all of the tests against each PR because it's **target** is our repo, regardless of where it is running? We need it to integrate into **our repo**, hence the integration tests. Head-scratcher.

Reply:
Nah, Netlify runs across forks. It's this particular GitHub action that doesn't.

Reply:
That's unfortunate because Netlify doesn't give you a preview link for each changed file, and the Github action does (even following includes to their output pages).

Reply:
Agreed.